lib/zipkin-tracer/trace.rb

```diff
@@ -0,0 +1,67 @@
+require 'finagle-thrift'
+require 'finagle-thrift/trace'
+
+module Trace
+  class Span
+    def to_h
+      {
+        name: @name,
+        traceId: @span_id.trace_id.to_s,
+        id: @span_id.span_id.to_s,
+        parentId: @span_id.parent_id.nil? ? nil : @span_id.parent_id.to_s,
```

Comment:
there's also :timestamp (microseconds, usually first annotation) and :duration (also microseconds, can be derived from last timestamp - first timestamp). A "complete" span should send these two keys.